google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITQueryTest.java

```diff
@@ -427,6 +445,37 @@ public void bindStringArrayNull() {
     assertThat(row.isNull(0)).isTrue();
   }
 
+  @Test
+  public void bindJsonArray() {
+    assumeFalse("Emulator does not yet support JSON", EmulatorSpannerHelper.isUsingEmulator());
+    Struct row =
+        execute(
+            Statement.newBuilder("SELECT @v").bind("v").toJsonArray(asList("{}", "[]", null)),
+            Type.array(Type.json()));
+    assertThat(row.isNull(0)).isFalse();
+    assertThat(row.getJsonList(0)).containsExactly("{}", "[]", null).inOrder();
+  }
+
+  @Test
+  public void bindJsonArrayEmpty() {
+    assumeFalse("Emulator does not yet support JSON", EmulatorSpannerHelper.isUsingEmulator());
+    Struct row =
+        execute(
+            Statement.newBuilder("SELECT @v").bind("v").toJsonArray(Collections.emptyList()),
+            Type.array(Type.json()));
+    assertThat(row.isNull(0)).isFalse();
+    assertThat(row.getJsonList(0)).containsExactly();
```

Comment:
nit: this assertion reads a bit funny, could you do an `assertThat(...).isEqualTo(Collections.emptyList())` instead?